Hippo.Email/Models/SimpleNotificationModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hippo.Email.Models
+{
+    public class SimpleNotificationModel
+    {
+        public string UcdLogoUrl { get; set; } = "https://hippo-test.azurewebsites.net/media/caes-logo-gray.png";
```

Comment:
Don't have it point to test.

Reply:
Copied that from elsewhere. Going to have to make a couple changes...